ilastik/applets/dataSelection/dataSelectionGui.py

```diff
@@ -413,13 +414,23 @@ def addFiles(self, roleIndex, startingLaneNum=None):
         paths = ImageFileDialog(self).getSelectedPaths()
         self.addFileNames(paths, startingLaneNum, roleIndex)
 
-    def addFileNames(self, paths: List[Path], startingLaneNum: int, roleIndex: int):
+    def addFileNames(self, paths: Optional[List[Path]], startingLaneNum: Optional[int], roleIndex: int):
         # If the user didn't cancel
-        for path in paths or []:
+        if paths is None:
+            paths = []
+
+        # startlingLaneNum == -1 if dragged onto the empty table.
+        if startingLaneNum is not None and startingLaneNum >= 0:
+            lane_indices = itertools.count(startingLaneNum)
+        else:
+            # startlingLaneNum = -1 if dragged onto the empty table
```

Comment:
```suggestion
```

Reply:
that comment I'd leave in because it's not obvious from code

Reply:
It's a duplicate; or you can delete the other one.